app/views/people/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Task Masters</h1>
+<hr />
+
+<div class="list">
+  <% @people.each do |p| %>
+      <div class="task">
+        <%= link_to p.name, "/people/#{p.id}" %>
+        <br>
+        Number of tasks to complete:
+        <% total_tasks = 0 %>
+        <% p.tasks.each do |task| %>
+          <% if task.completed_at.nil? %>
+          <% total_tasks += 1 %>
+          <% end %>
+        <% end %>
```

Comment:
Because lines 10-15 are all ruby code and no HTML, you can actually write this as one big ERB tag over multiple lines:
```erb
<%
  total tasks = 0
  p.tasks.each do |task|
    if task.completed_at.nil?
      total_tasks += 1
    end
  end
%>
```